sweepbatcher/sweep_batcher_test.go

```diff
@@ -1300,24 +1429,29 @@ func TestHandleSweepTwice(t *testing.T) {
 			return false
 		}
 
+		// Find the batch with largest ID. It must be the second batch.
+		var secondBatch *batch
+		for _, batch := range batches {
+			if secondBatch == nil || batch.id > secondBatch.id {
```

Comment:
interesting way to find it. Little hard to read imo, could we
```
secondBatch := batches[0]
if batches[1].id > batches[0].id {
        secondBatch = batches[1]
}
```

Reply:
Variable batches is a map, not a slice, and we don't know the keys (in practice 0,1 or 1,2) so we have to visit all the items and find the one with maximum id.